events/eventclass.py

```diff
@@ -15,10 +16,17 @@ def equal_times(time_1, time_2):
     Datetime objects are rounded to the nearest millisecond before comparison.
     Used for comparing Event objects, since MongoDB truncates times to the
     nearest millisecond.
+
+    Strings representing times are directly compared for equality.
     """
-    time_1 = time_1.replace(microsecond=(time_1.microsecond // 1000) * 1000)
-    time_2 = time_2.replace(microsecond=(time_2.microsecond // 1000) * 1000)
-    return time_1 == time_2
+    if type(time_1) is type(time_2):
+        if isinstance(time_1, datetime.datetime):
+            time_1 = time_1.replace(
+                microsecond=(time_1.microsecond // 1000) * 1000)
+            time_2 = time_2.replace(
+                microsecond=(time_2.microsecond // 1000) * 1000)
+        return time_1 == time_2
+    return False
```

Comment:
If we're doing time as strings, why do you need this replacing still? Are you planning on also supporting datetimes at the same time?

Reply:
I guess I wanted to leave functionality there in case times had to be compared as datetimes in the backend, but I don't think it's used anywhere right now so I can remove it.